### ISSUE TITLE
Unlocked Grip+CGO dodge animation not working Properly!

### ISSUE THREAD

**Issue by RandomUser5251** [ShikyoKira/Project-New-Reign---Nemesis-Main#152]
Just as the Title Say's after installing your mod. the Dodge function of Unlocked Grip+CGO no longer works properly. as in when pressing the dodge key, the character no longer moves to the direction it's moving into. but instead the animation plays but the character is still, stuck on the same position and not moving.

And also don't even try recommending the use of MO2. that mod manager caused a lot more Trouble and problems than it could solve.

**Comment by ShikyoKira** [comment]
upload your animationdatasinglefile.txt thats located in data/meshes

**Comment by RandomUser5251** [comment]
[Errors and More Errors!.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-

[animationdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/4194465/animationdatasinglefile.txt)

[animationsetdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/4194471/animationsetdatasinglefile.txt)


[Uploading nemesis_animationdatasinglefile.txt…]()


[nemesis_animationsetdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/4194475/nemesis_animationsetdatasinglefile.txt)

**Comment by ShikyoKira** [comment]
Does this problem persist if you only have CGO without Nemesis?

Do this test, reinstall CGO, place it at the bottom in your left panel. Run skyrim and test. Update me the result

**Comment by RandomUser5251** [comment]
no it doesn't and iv'e been trying to figure it out for about a week now.

**Comment by ShikyoKira** [comment]
Surely there is some mistakes somewhere. If it doesn't work even without Nemesis, means the problem exists out of Nemesis, something beyond my control. There might be mod conflict. Could it be that you still have FNIS installed?

**Comment by RandomUser5251** [comment]
Unlocked grip+Cgo works dodging mechanic works without nemesis. but with nemesis installed the dodge mechanic stops working properly the same can also be said with tk-dodge.  and believe me when i say it doesn't work since iv'e been trying to find away to make it work. for almost two weeks now. and i assure you i am not the only one having this problem.

**Comment by ShikyoKira** [comment]
if i can't recreate the problem, its near impossible for me to fix it as i don't know whats the cause. It will be working in the dark for me

I just gave a thorough check of those files you sent me. Nemesis uses the same read and write format but it failed to read the file you sent me. I even manually gone through it to confirm what's wrong. Its definitely not from Nemesis for there are many missing data and incorrect placement of data

Send me your patchlog.txt located in `data/nemesis_engine`

> and i assure you i am not the only one having this problem

Well, so far, 10 out of 10 cases i get ended up the cause is on their side not mine so far. While I don't want to dismiss the possibility that it might be a new bug, you will have to provide me more info.

Alright, tell me your minimum setup (minimum load order), your mod manager if you have any, your skyrim version (LE/SE and version). Start from installing the mod, running all the configurations to where and when you experience problematic dodge in a new game

I'll do exactly as the info you provide, see if i can recreate the same issue

**Comment by RandomUser5251** [comment]
[loadorder.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/4209007/loadorder.txt)

This is my Current load order without Nemesis.  and i already tried starting a new game a while back and that didn't work either. my only clue why dodge function stopped working is because of the Behaviour.hkx file.

**Comment by ShikyoKira** [comment]
I noticed you have FNIS_PCEA2 installed, Nemesis doesn't support that, we have our own [PCEA](https://www.nexusmods.com/skyrim/mods/100942). Do you have FNIS installed as well?

**Comment by RandomUser5251** [comment]
the reason i had pcea, is this is my current load order without Nemesis installed. also i already tried the PCEA you had before going to this site, and it still had the same results. the custom animations work but the dodge mechanic, does not work properly.

**Comment by ShikyoKira** [comment]
sorry for the late respond. Got busy lately. Anyway, the load order is too long for me to follow. Since you aren't using MO2, i don't even know your installation order. It might be too much for you but disable all your mods but the CGO and Nemesis and test it, we do this all the time when trying to diagnose the problem (however, we use MO2 to do that) so I hope you have your way to do that

**Comment by RandomUser5251** [comment]
i use NMM to install most of my mods. already did that as well, the results are the same.

**Comment by Ingerasul24** [comment]
> Just as the Title Say's after installing your mod. the Dodge function of Unlocked Grip+CGO no longer works properly. as in when pressing the dodge key, the character no longer moves to the direction it's moving into. but instead the animation plays but the character is still, stuck on the same position and not moving.
> 
> And also don't even try recommending the use of MO2. that mod manager caused a lot more Trouble and problems than it could

**Comment by Ingerasul24** [comment]
I had the same problem like you ...to fix it you need to delete all your animations and behavior files and fnis, then redownload all and run nemesis ...and because fniss for some wierd reasons leaves a trace in your saves somehow you need to load a save before you instaled fnis and then everything is gonna work smoothly...bdw some animatitons from CGO are weird..is not nemesis fault

**Comment by RandomUser5251** [comment]
> I had the same problem like you ...to fix it you need to delete all your animations and behavior files and fnis, then redownload all and run nemesis ...and because fniss for some wierd reasons leaves a trace in your saves somehow you need to load a save before you instaled fnis and then everything is gonna work smoothly...bdw some animatitons from CGO are weird..is not nemesis fault

then explain TKDoddge not working.

**Comment by ShikyoKira** [comment]
i don't think @Ingerasul24 or I can even explain it. Sometimes missing file(s) just happen, we don't know what you have installed or uninstalled, so with things in and out, something missing is bound to happen and as Ingerasul24 said, sometimes we just have to reinstall mods to fix it

**Comment by Ingerasul24** [comment]
I have CGO and AGO and for some time i had ultimate dodge and they all worked like they supose too but before all this i had instaled fnis because one of the mods had it as a master and that created a lot of problems i tried to restart the game multiple times and sometimes CGO worked sometimes not . What i did is what ive said in my last comment and since then i had no problem with it .

**Comment by Ingerasul24** [comment]
Ps ..to solve the issue with the mod that had fnis as a master ive created an empty plugin and name it fnis and that's it

**Comment by ShikyoKira** [comment]
oh i got another report and after testing with him, we suspect it might be the work of an SKSE plugin, as far as i know, the insertAttackData.dll from UC does that, so perhaps there might be other mod does the same thing. Do check it out

**Comment by RandomUser5251** [comment]
> Ps ..to solve the issue with the mod that had fnis as a master ive created an empty plugin and name it fnis and that's it ,and im using MO2 ,is making thing's easier...I'm leaving all this comments cause maybe is gonna help someone with similar problem ..

tried your method and it didn't work at all it still has the same BS problem, that it has before even tried using the new version of CGO and still the same.

**Comment by xFrosty7** [comment]
@RandomUser5251 dude this is getting rediculous, why is this issue not getting attention? i literally just need dodging to work with fnis and my game is complete ffs.

**Comment by Ingerasul24** [comment]
Dude nemesis and fnis basically do the same thing ..drive a car and ride a bicycle at the same time...if you have fnis and nemesis instaled you asking for trouble

**Comment by Ingerasul24** [comment]
This mod works with nemesis..and soon is gonna be upgraded...use mod organizer2 to install your mods ..delete fnis delete your saves delete overwrite folder install nemesis run update engine wait to finish run launch nemesis behavior engine and wait to finish ..now you can play if you don't install your mods with mo2 you're in for a headache ...in that situation you have to delete all the behaviors generated by the fnis and if you are lucky your animation files to be intact ..have fun

**Comment by xFrosty7** [comment]
@Ingerasul24 i typed fnis by mistake, i meant nemesis and i'm not new to this.
first of all, the problem is only in 3rd person, in 1st person it works fine.
i've already deleted fnis it's mods, i tried disabling all my mods excluding TUDM and Nemesis, emptying overwrite folder (mo2), running nemesis generator with the patch, still doesn't work.
i did everything i could, i realized that for some people it works and for some it doesn't, and nemesis's author isn't helping much sadly :/

**Comment by ShikyoKira** [comment]
drop me your discord username. We will further discuss there in details. Since none of my beta testers are having this issue, i'm interested in hearing about your issues seeing that you have seemingly tried your best in fixing this

**Comment by jukicjepeder** [comment]
using mo2, had the same issue
right click ''overwrite'' in load order > clear all > started working
idk how, good luck

**Comment by Crestfall69** [comment]
Got the same issue. What happened?

**Comment by RandomUser5251** [comment]
the mod author doesn't give a damn about this problem and tends to blame it on other mods. he has been doing this since January, and still refuses to help fix the problem it' is also not helping when a bunch of self entitled pricks Especially the Premium members on nexus keeps praising this Garbage of a mod. and tends to act violently when the problem is being pointed out and not being fixed. seriously to the mod author either fix your mod, and stop blaming other's work when what you made is clearly causing the issue. or Delete your Damn! mod from the nexus.,

**Comment by Ingerasul24** [comment]
Why do you use the mod if it's not working ? Stop fucking complaining so much and don't use the mod for something that is free and requires a lot of work you a fucking dumb ass .u don't like it don't llay with it ..delete skyrim ..moding is not for you play mario

**Comment by Ingerasul24** [comment]
U getting a god damn house for free and you complayn about the fucking decoration ..delete skyrim an al those fucking folders and saves instal this mod and cgo only and if it's working as intended ur a dick head if is not this mod is to complicated for you

**Comment by RandomUser5251** [comment]
Really Asshole why dont you keep sucking on this shitty mod. that doesn't even work properly i've been using mods for skyrim for years now, and this is the only mod that refuses to work properly, i tried reinstalling the mod and doing a Clean Reinstall with this stupid mod only and it still doesn't work properly.

**Comment by xFrosty7** [comment]
@Ingerasul24 lol are you retarded?

**Comment by xFrosty7** [comment]
@Ingerasul24 i do appreciate that mods are free and know that they require lots of work, this isn't about "if you don't like it dont use it", the only reason i'm using this mod is because it is required by mods such as cgo and other mods and it's causing people issues, so technically it's not working as intended.

**Comment by RandomUser5251** [comment]
> @Ingerasul24 i do appreciate that mods are free and know that they require lots of work, this isn't about "if you don't like it dont use it", the only reason i'm using this mod is because it is required by mods such as cgo and other mods and it's causing people issues, so technically it's not working as intended.

Finally someone who understands the Situation. that the problem is with this mod. is not working as intended.  with either CGO TUDM and Tk Dodge not working properly with this mod active.  since i have already tried them hundreds of times. and the Same Results as always the 3rd person dodge isn't moving the animation plays but the player stays in the same position. and as a Modder and a Mod User i can understand when some people complain on something you work hard on criticize them. but a mod that causes other mods to not work properly when installed and is also required, but ends up breaking the other one, and causing more problem than it could solve that's when others like me will start to complain and get agitated, especially if the creator of the said mod. callously ignores the problem and blames it on the other. and whats worst is that some users mod reviewers tend to over hype  or over sell the mod. making the people who are asking for a solution be ignored. Just because Nemesis can handle more animations does not make it better.

**Comment by whitewolf2899** [comment]
@RandomUser5251 There is a reason y it is still in "Beta" respect that wait till it get fixed and STFU.. there are many issues to deal with and if u don't want it go and install FNIS what can I say more.

**Comment by GERexperience** [comment]
I faced the same problem.
And tried to solve it in all the ways described with the author's recommendations. such as deleting InsertAttackData.dll and other things like deleting all FNIS files clearing them in Behavior and Animations disabling mods and so on. But nothing helped. Now I'm trying to clear the rest of the Skse plugins in the hope that this error will not occur after updating the animations in Nemesis. I need a CGO to complete the build. Later I will try to put on a clean game that is only CGO and Nemesis but I think nothing will change. Has anyone been able to solve the problem yet ?

**Comment by GERexperience** [comment]
I noticed that the mod works (the dodge works fine) if you replace the file in the mesh - animationdatasinglefile.txt from the TsGO mod and other dodging mods. And do not generate nemesis. But all other animations do not work. Actually, I deleted all the animated mods except CGO as well as some skse plug-ins, and generated a nemesis - the problem remained as it was dodged in one place. In the future, I want to check on another computer with Windows 10 on my Windows 7, I think maybe the program does not work correctly because of Windows .... I can’t say for sure, but I'm pretty sure that my nemesis does not want to work correctly with Dodge mods.

**Comment by RandomUser5251** [comment]
> I noticed that the mod works (the dodge works fine) if you replace the file in the mesh - animationdatasinglefile.txt from the CGO mod and other dodging mods. And do not generate nemesis. But all other animations do not work. Actually, I deleted all the animated mods except CGO as well as some skse plug-ins, and generated a nemesis - the problem remained as it was dodged in one place. In the future, I want to check on another computer with Windows 10 on my Windows 7, I think maybe the program does not work correctly because of Windows .... I can’t say for sure, but I'm pretty sure that my nemesis does not want to work correctly with Dodge mods.

I tried doing that as well replacing that text file. and the results are the same, the dodge animation plays but it doesn't move and is, worst part doing that will sometimes crashes the game instantly,  even did a full clean reinstall of the game deleted every single file and have cgo and this mod installed. and the same problem persists as i even tried it with tk dodge Tudm and such a hundred times. and i have been trying to make this work since February by the way. 

and as for whitewolf2899 why dont you Shut the Hell up i know it's in beta, but the fact that a lot of mods that i need are now requiring this broken mod. is not making it any better as they are forcing a broken mod to be used with theirs, when they couldn't even give a proper solution when asked and tends to act high and mighty, by ignoring the problem saying that it isn't happening on them not to mention people like you are not even helping at all. with your ignorance.

**Comment by RandomUser5251** [comment]
> I noticed that the mod works (the dodge works fine) if you replace the file in the mesh - animationdatasinglefile.txt from the CGO mod and other dodging mods. And do not generate nemesis. But all other animations do not work. Actually, I deleted all the animated mods except CGO as well as some skse plug-ins, and generated a nemesis - the problem remained as it was dodged in one place. In the future, I want to check on another computer with Windows 10 on my Windows 7, I think maybe the program does not work correctly because of Windows .... I can’t say for sure, but I'm pretty sure that my nemesis does not want to work correctly with Dodge mods.

I Tried your Method just now, and for some odd Reason it Worked. and CGO's Dodge Feature is now Working as it, is intended to you and XFrosty7 Deserves a proper Thanks as you two are the only who seem's to care about this problem, when the mod author himself Refuses to provide any proper solution to the problem not to mention, People  like whitewolf2899 and Ingerasul24 aren't helping due to how blind they are just because theres a new shiny mod claiming to be better than it's predecessors. So thanks to the both of you.

**Comment by GERexperience** [comment]
> > I noticed that the mod works (the dodge works fine) if you replace the file in the mesh - animationdatasinglefile.txt from the CGO mod and other dodging mods. And do not generate nemesis. But all other animations do not work. Actually, I deleted all the animated mods except CGO as well as some skse plug-ins, and generated a nemesis - the problem remained as it was dodged in one place. In the future, I want to check on another computer with Windows 10 on my Windows 7, I think maybe the program does not work correctly because of Windows .... I can’t say for sure, but I'm pretty sure that my nemesis does not want to work correctly with Dodge mods.
> 
> I Tried your Method just now, and for some odd Reason it Worked. and CGO's Dodge Feature is now Working as it, is intended thanks to you and XFrosty7 Deserves a proper Thanks as you two are the only who seem's to care about this problem, when the mod author himself Refuses to provide any proper solution to the problem not to mention, People like whitewolf2899 and Ingerasul24 aren't helping due to how blind they are just because theres a new shiny mod claiming to be better than it's predecessors. So thanks to the both of you.

I am glad that this method helped you and you can use the CGO mod. However, unfortunately, other animation mods stop working correctly - since there is no animation. It all comes down to the problem of not being able to use this mod and Nemesis to generate other animations... unfortunately, I still don't understand why this happens. After the test on a different version of windows, I'll see what's going on. Because someone's animation generation is successful and the CGO is working. and there are those like me who even on the clean version there is a roll on the spot with any of the roll mods

**Comment by RandomUser5251** [comment]
> > > I noticed that the mod works (the dodge works fine) if you replace the file in the mesh - animationdatasinglefile.txt from the CGO mod and other dodging mods. And do not generate nemesis. But all other animations do not work. Actually, I deleted all the animated mods except CGO as well as some skse plug-ins, and generated a nemesis - the problem remained as it was dodged in one place. In the future, I want to check on another computer with Windows 10 on my Windows 7, I think maybe the program does not work correctly because of Windows .... I can’t say for sure, but I'm pretty sure that my nemesis does not want to work correctly with Dodge mods.
> > 
> > 
> > I Tried your Method just now, and for some odd Reason it Worked. and CGO's Dodge Feature is now Working as it, is intended thanks to you and XFrosty7 Deserves a proper Thanks as you two are the only who seem's to care about this problem, when the mod author himself Refuses to provide any proper solution to the problem not to mention, People like whitewolf2899 and Ingerasul24 aren't helping due to how blind they are just because theres a new shiny mod claiming to be better than it's predecessors. So thanks to the both of you.
> 
> I am glad that this method helped you and you can use the CGO mod. However, unfortunately, other animation mods stop working correctly - since there is no animation. It all comes down to the problem of not being able to use this mod and Nemesis to generate other animations... unfortunately, I still don't understand why this happens. After the test on a different version of windows, I'll see what's going on. Because someone's animation generation is successful and the CGO is working. and there are those like me who even on the clean version there is a dodge on the spot with any of the dodge mods
> -P.S I would like to ask what kind of windows PC you have?

i have a 64-bit  Windows 7 Ultimate Pc, using Pentium (R) Dual core. e 5700 @ 300 Ghz. 

and By the way Thanks again for the Help i Really Appreciate it since you and Xfrosty7 who are the ones who can understand the situation as well as fully aware of this problem. unlike the one's on Nexus especially the one's who have premium on their user name. who did'n t do anything to help solve alleviate the problem.

and by the way this might be a long shot to fix your problem. but had you tried putting Nemesis engine first manually, then installing the Reign plugin via NMM. i dont use Vortex or that MO2 since it caused me a lot of problems when i used those two. and then installing CGO, and then adding the animation files you want, before updating and launching the engine, and then replacing the text file. that you mentioned with the one from cgo. and by the way i did find something interesting in the animationdatasinglefile.txt from both CGO and the one the engine generates. 
as in the CGO's textfile has the proper coding for the Dodge Function to work. but when you check the one that, the Engine Generates has a bunch of mismatched/Random coding for the dodging mechanic.  causing it to not move when being used.

**Comment by GERexperience** [comment]
> and by the way i did find something interesting in the animationdatasinglefile.txt from both CGO and the one the engine generates.
> as in the CGO's textfile has the proper coding for the Dodge Function to work. but when you check the one that, the Engine Generates has a bunch of mismatched/Random coding for the dodging mechanic. causing it to not move when being used.

This is why the program does not generate the data as it should. By the way, I also noticed that one of the examples of the Dodge Mod-the ultimate Dodge Mod absolutely refuses to work... even hints of animation and endurance costs. However, if you throw the txt file from the mod and this mod works fine.(again without animations from other mods) I wonder if the author of Nemesis will respond again ? - I already did a clean build of Nemesis + CGO but I was left with the same problem.. Why do I generate everything incorrectly I still suspect it is windows or some supernatural forces that do not allow me to complete my game build)))

**Comment by RandomUser5251** [comment]
> This is why the program does not generate the data as it should. By the way, I also noticed that one of the examples of the Dodge Mod-the ultimate Dodge Mod absolutely refuses to work... even hints of animation and endurance costs. However, if you throw the txt file from the mod and this mod works fine.(again without animations from other mods) I wonder if the author of Nemesis will respond again ? - I already did a clean build of Nemesis + CGO but I was left with the same problem.. Why do I generate everything incorrectly I still suspect it is windows or some supernatural forces that do not allow me to complete my game build)))

 it's Definitely not related to the windows os. but it's definitely related to the mod Engine. and how it generates the said file, since when updating or generating the Behavior files, it tends to crash multiple times a problem that is very constant when using it, 

and i highly doubt the mod author would even care, or respond at all seeing as he has constantly ignored this problem for almost 5-months now. and claims it's not his mod that, that is the source of the problem. but when it's clear that it is the one causing it due to mismatched/wrong coding of the text file. that it generates. 

not to mention some mod reviewers in youtube  tend to overhyped or oversell the mod. to the point that they think like it couldn't do any wrong. which is why i have zero enthusiasm on The Fallout:Frontier mod. which everyone is hyped about.

**Comment by Ingerasul24** [comment]
All i hear is nemesis ...if cgo is the actual problem since it toucher a verry sensitive behavior files and its fucking up some animations ..why don't you ask the guy who made cgo about forward- left and f- r power animations ...how about npc animations? Btw in the old skyrim nexus you can find cgo that works with fnis ..try that one ...and after that start complain about fnis

**Comment by Ingerasul24** [comment]
Maybe there is something wrong with this mod ..but so far the only problem I've had with it it was when i had fnis instaled ...this is my 3rd full gameplay with it ..and i have no problem ...i deleted cgo since is a mess ...but im playing with ultimate dodge ..dynamic animations ..and some animations replacers ..potion animated fix ....but either way my first full gameplay it was with cgo and ago they are nice mods but they don't work with other animations and they have a lot of conflicts in behavior even between them

**Comment by Ingerasul24** [comment]
I don't understand why so much stupidity exist on this planet ...why the fuck you keep praising a buggy mod ?? Like ago and cgo till the last update the fucking mod didn't work well with crossbows and it still doesn't work well with power animations...plus is very fucking sensitive and conflicting with every fucking animation mod that exists on nexus without a patch ...but yea your to stupid to see that ...and if you have do delete dataanimationsinglefile that means you have a conflict..but yea this is what it means to be stupid and proclaim that you know what you doing...im not gonna waste my time with you ..my last mesage

**Comment by GERexperience** [comment]
> All i hear is nemesis ...if cgo is the actual problem since it toucher a verry sensitive behavior files and its fucking up some animations ..why don't you ask the guy who made cgo about forward- left and f- r power animations ...how about npc animations? Btw in the old skyrim nexus you can find cgo that works with fnis ..try that one ...and after that start complain about fnis

Unfortunately, not only with the CGO problem, but also with EL dodge and Ultimate dodge - which does not work at all
You say there was a FNIS version of CGO for some reason I don't find one

**Comment by GERexperience** [comment]
> > This is why the program does not generate the data as it should. By the way, I also noticed that one of the examples of the Dodge Mod-the ultimate Dodge Mod absolutely refuses to work... even hints of animation and endurance costs. However, if you throw the txt file from the mod and this mod works fine.(again without animations from other mods) I wonder if the author of Nemesis will respond again ? - I already did a clean build of Nemesis + CGO but I was left with the same problem.. Why do I generate everything incorrectly I still suspect it is windows or some supernatural forces that do not allow me to complete my game build)))
> 
> it's Definitely not related to the windows os. but it's definitely related to the mod Engine. and how it generates the said file, since when updating or generating the Behavior files, it tends to crash multiple times a problem that is very constant when using it,
> 
> and i highly doubt the mod author would even care, or respond at all seeing as he has constantly ignored this problem for almost 5-months now. and claims it's not his mod that, that is the source of the problem. but when it's clear that it is the one causing it due to mismatched/wrong coding of the text file. that it generates.
> 
> not to mention some mod reviewers in youtube tend to overhyped or oversell some mods. to the point that they think like it couldn't do any wrong. and when someone complains about it they ganged up on him/her to the point of Deleting their comments so no one can read about the problems with the said mod. the same goes on Nexus especially the one' who has Premium underneath their user name, these guys tend to act high and mighty and when someone asks for help they make it seem like a joke and would act as a complete Jerkm in which further agitates the person asking for help. and bans the said person after they provoked him/her with their stupid retorts and Replies.
> 
> which is why i have zero enthusiasm on The Fallout:Frontier mod. which everyone is hyped about. since those same people would do it again and again.

I did a little test, downloaded TK dodge and generated it with Nemesis - and got the roll in place. After that, I downloaded Fnis and generated it with the same TKdodge - everything was fine there and the mod worked as expected considering my not a large number of mods for animation. So same Nemesis in my case on my Assembly generates 6к animations and Fnis about 4к not know plays whether this any role.
This is probably my last post. And I will have to give up on this problem because I do not know how to fix it by sitting over it for several hours a day. A place to enjoy the game

**Comment by RandomUser5251** [comment]
> I did a little test, downloaded TK dodge and generated it with Nemesis - and got the roll in place. After that, I downloaded Fnis and generated it with the same TKdodge - everything was fine there and the mod worked as expected considering my not a large number of mods for animation. So same Nemesis in my case on my Assembly generates 6к animations and Fnis about 4к not know plays whether this any role.(Not game role)
> This is probably my last post. And I will have to give up on this problem because I do not know how to fix it by sitting over it for several hours a day. A place to enjoy the game

Good for you, it's best to stick with FNIS since it has proven to work properly time and time again, and easy to use, despite the some of it's Limitations.

**Comment by Ingerasul24** [comment]
https://1drv.ms/v/s!AssbZ0B-CT51gYI6WYrNRVvX_KyxZg?e=b7mh6f    sorry for the quality and framerate ..is my first video but that's not the focus of the video .enjoy

**Comment by ShikyoKira** [comment]
Finally. Too much toxicity here

@RandomUser5251 first of all, I got 50+ issues here to deal with. Your issue is something that I can't recreate, hence, something I can't fix. I got more issues to deal with obvious I am focusing on issues that I can recreate and analyze. You are being disrespectful here. If you don't want to you use Nemesis, go use FNIS. No one is forcing you

I have spent 3000 hours on developing this tool. You spending a mere few days on it is nothing comparable

The bottom line is if this doesn't work for you and I can't fix it FOR YOU (not that I don't want to fix it) yet it works for me, then move on. I tried my best for you and I got more people to serve not just you, don't be selfish
